src/packer.pkr.hcl

```diff
@@ -131,8 +131,15 @@ build {
   }
 
   provisioner "shell" {
-    execute_command = "chmod +x {{ .Path }}; sudo env {{ .Vars }} {{ .Path }} ; rm -f {{ .Path }}"
     script          = "src/post_setup.sh"
+    execute_command = "{{ .Vars }} bash '{{ .Path }}'"
+    skip_clean      = true
```

Comment:
These keys should be re-alphabetized.

Reply:
Is this what you mean: [d9dfc6a](https://github.com/cisagov/ubuntu-server-packer/commit/d9dfc6a0c14597000451af69de878a17c36d7ae9)

Reply:
Yes, that is what I meant.